### PR TITLE
Update euclid.

### PR DESCRIPTION
**Pull request by emilio**

### CODE REVIEW

**Comment by emilio** [comment]
r? anyone (@jdm @SimonSapin)

File: surfman/Cargo.toml [hunk 1]
```diff
@@ -36,7 +36,7 @@ log = "0.4"
 parking_lot = "0.10.2"
 
 [dependencies.euclid]
-version = "0.20"
+version = ">=0.20,<=0.22"
```
Comment (jdm) [inline surfman/Cargo.toml:39]: Why the range?

**Comment by jdm** [comment]
r=me

File: surfman/Cargo.toml (hunk 1 above)
Comment (emilio) [reply surfman/Cargo.toml:39]: Because there's a lot of other stuff to update basically :-)

**Comment by atouchet** [comment]
Can the Android example dependency be updated as well like in #201?

**Comment by asajeffrey** [comment]
I think #234 can close this?

**Comment by mrobinson** [comment]
This was done in #244.

**Pull request closed without merging**